README.rst

```diff
@@ -1,21 +1,32 @@
-``zope.hookable``
-=================
+===================
+ ``zope.hookable``
+===================
 
 .. image:: https://img.shields.io/pypi/v/zope.hookable.svg
-    :target: https://pypi.python.org/pypi/zope.hookable/
-    :alt: Latest Version
+		:target: https://pypi.python.org/pypi/zope.hookable/
+		:alt: Latest release
+
+.. image:: https://img.shields.io/pypi/pyversions/zope.hookable.svg
+		:target: https://pypi.org/project/zope.hookable/
+		:alt: Supported Python versions
 
 .. image:: https://travis-ci.org/zopefoundation/zope.hookable.png?branch=master
         :target: https://travis-ci.org/zopefoundation/zope.hookable
 
 .. image:: https://readthedocs.org/projects/zopehookable/badge/?version=latest
-        :target: http://zopehookable.readthedocs.org/en/latest/
+        :target: http://zopehookable.readthedocs.io/en/latest/
         :alt: Documentation Status
 
+.. image:: https://coveralls.io/repos/github/zopefoundation/zope.hookable/badge.svg?branch=master
+        :target: https://coveralls.io/github/zopefoundation/zope.hookable?branch=master
+
+
```

Comment:
They're visually aligned in the file (at least for me). 

What happened was that I copied the first two badges directly out of the ZODB readme, which used a different alignment than the fields here. When I corrected that, they accidentally ended up with a tab character instead of spaces 😢